cmd/templates.go

```diff
@@ -66,10 +66,10 @@ mirror:
         - name: cluster-logging
           channels:
             - name: 'stable'
-  {{- if eq .Channel "4.12" }}
+  {{- if VersionAtLeast .Version "4.10" }}
         - name: odf-lvm-operator
```

Comment:
The odf-lvm-operator is replaced by lvms-operator in 4.12

Reply:
Handled in new commit